Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -141,10 +142,29 @@ function prepareMergeConcept(conceptId) {
     if(conceptsToMerge != '') {
         conceptsToMerge = conceptId;
     } else {
-        conceptsToMerge = conceptsToMerge + "," + conceptId;	
+        conceptsToMerge = conceptsToMerge + "," + conceptId;    
     }
 }
 
+var createWrapper = function(conceptId) {
+  $.ajax({
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json"
+      },
+      type : "GET",
+      url : "${pageContext.servletContext.contextPath}/rest/Concept",
+      data : {
+        id : conceptId
+      },
+      success : function(details) {
+        $.each(details.conceptEntries, function (index, conceptEntry) {
+          window.location = '${pageContext.servletContext.contextPath}/auth/conceptlist/addconceptwrapper?word=' + conceptEntry.lemma + '&selectedConceptList=' + conceptEntry.conceptList + '&description=' + conceptEntry.description + '&selectedType=' + conceptEntry.type + '&wrapperids=' + conceptEntry.id;
+        });
```

Comment:
I'm pretty sure you have all this info when you build the results page, why do you make an extra concept request?